models/_sector/nft/trades/nft_trades_beta.sql

```diff
@@ -0,0 +1,13 @@
+{{ config(
+    schema = 'nft',
+    alias = 'trades_beta',
+    partition_by = ['blockchain','project','block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
```

Comment:
can you add incremental predicates here too? to refine the target read, similar to addition to macro for source read?